.github/config.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project. Please make sure not to start working on the issue, unless you get assigned to it.😄
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request. 🙌🙌 We will get back to you as soon as we can 😄.
+  
+  
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉 All the best for your amazing open source journey ahead 🚀.
+  
```

Comment:
Please add a new line here, helps git perform its magic.

Reply:
@RAJAGOPALAN-GANGADHARAN  you want to go them one below the other?

Reply:
have resolved it